libtenzir/builtins/functions/ip.cpp

```diff
@@ -0,0 +1,85 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2024 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "tenzir/ip.hpp"
+
+#include "tenzir/concept/parseable/tenzir/ip.hpp"
+#include "tenzir/type.hpp"
+
+#include <tenzir/arrow_utils.hpp>
+#include <tenzir/concept/parseable/tenzir/time.hpp>
+#include <tenzir/series_builder.hpp>
+#include <tenzir/table_slice_builder.hpp>
+#include <tenzir/tql2/plugin.hpp>
+
+#include <arrow/array/array_base.h>
+#include <arrow/array/array_binary.h>
+#include <arrow/array/array_nested.h>
+#include <arrow/array/builder_nested.h>
+#include <arrow/type_fwd.h>
```

Comment:
Not that it matters too much, but perhaps you can change the `"..."` to `<...>`. And you probably don't need any of the includes at the bottom.

Reply:
Auto-includes strike again; I cleaned them up.
I also added a `check()` call to the builder result.